src/comment/comment.resolver.ts

```diff
@@ -0,0 +1,28 @@
+// comment.resolver.ts
+import { Resolver, Query, Mutation, Args, ID } from '@nestjs/graphql';
+import { Comment } from './comment.model';
+import { CommentService } from './comment.service';
+import { CreateComment } from './comment.dto';
+import { Job } from '../job/job.model';
+
+
+@Resolver(() => Comment)
+export class CommentResolver {
+    constructor(private readonly commentService: CommentService) { }
+
+    // add the create to resolver
+    @Query(() => [Comment])
+    async findAll(): Promise<Comment[]> {
+        return this.commentService.findAll();
+    }
+
+    @Query(() => [Comment])
+    async findByJob(@Args('jobID', {type: () => ID}) jobID: string): Promise<Comment[]> {
```

Comment:
NIT (short for nitpick, when you see this you can take it as a suggestion that isn't critical): A more user friendly approach could be to take in job as an ID, pass it through the Job pipe, and then search using the Job object.

The benefit is that the user of the GraphQL interface will know the difference between getting no comments because there aren't comments on the Job vs getting no comments because they didn't pass in a valid job ID. The Job pipe would give the user an error if the Job ID is invalid which can make debugging on the client side easier.

Alternatively, we can just remove this endpoint since the only way we should get comments is off a Job model which is already handled on the Job resolver.

Reply:
Decided to remove this query since we already have a job by id feature, and adding another query like this would be redundant. Thanks for pointing this out!